components/script/script_thread.rs

```diff
@@ -181,6 +185,8 @@ impl InProgressLoad {
             is_visible: true,
             url: url,
             origin: origin,
+            navigation_start: 0,
+            navigation_start_precise: 0.,
```

Comment:
I am not sure what you mean with storing the values here, they are already stored in `InProgressLoad` during the handling of the constellation message to script that notifies about the start of the navigation. Anyway, I agree that the constellation does not need to be aware of the timing code, so I am removing that part. Thanks!